repositories.bzl

```diff
@@ -122,6 +131,12 @@ def jazzer_dependencies():
         url = "https://repo1.maven.org/maven2/org/ow2/asm/asm-tree/9.4/asm-tree-9.4.jar",
     )
 
+    maybe(
+        http_jar,
+        name = "com_github_jsqlparser_jsqlparser",
+        url = "https://repo1.maven.org/maven2/com/github/jsqlparser/jsqlparser/4.4/jsqlparser-4.4.jar",
```

Comment:
While we're on it, there is a new version (4.5) of jsqlparser (https://repo1.maven.org/maven2/com/github/jsqlparser/jsqlparser/4.5/jsqlparser-4.5.jar). Can we use it?

Reply:
Done, also added the hash.